project/app/api/submission.py

```diff
@@ -44,15 +44,14 @@ async def submission_text(sub: Submission):
         try:
             # assert that the has is the same as the one passed with the file
             # link
-            assert hash.hexdigest() == sub.Pages[page_num]['Checksum']
+            assert hash.hexdigest() == sub.Pages[page_num]["Checksum"]
         except AssertionError:
             # return some useful information about the error including what
             # caused it and the file affected
-            return JSONResponse(status_code=422,
-                                content={
-                                    "ERROR": "BAD CHECKSUM",
-                                    "file": sub.Pages[page_num]
-                                })
+            return JSONResponse(
+                status_code=422,
+                content={"ERROR": "BAD CHECKSUM", "file": sub.Pages[page_num]},
+            )
         # add the response from google_api to a string with an ending
         # line break and the confidence flag from the method that determines if
         # the student is reminded about their handwritting
```

Comment:
I know these comments aren't part of this commit, but just noticing I'm a little confused by them, whether they're documentation or notes for future changes? If documentation, can you just clarify a bit? And perhaps remove/edit the "that determines if..." part of the sentence, since that's not fully implemented, and might cause confusion?